### PR TITLE
SONAR-9403 Add custom graph to the project activity page

### PR DESCRIPTION
**Pull request by gregaubert**

### CODE REVIEW

**Comment by gregaubert** [comment]
@stas-vilchik Tell me if you don't want to review this, since you couldn't review a big part of the sprint.

**Comment by gregaubert** [comment]
@stas-vilchik Thanks! I'll fix the issues that you found, thanks.
Yes the last graph that you selected or created is remembered in the localstorage and we show this one too on the project dashboard.
Yes that's correct, for now since we have only one custom graph available, as soon as you select a metric the other types are filtered out. I'll add a message to explain that to the user.

**Comment by gregaubert** [comment]
@stas-vilchik I applied some of the feedback and added two commits for the remaining two tickets that were not done yet.

File: server/sonar-web/src/main/js/apps/issues/sidebar/CreationDateFacet.js [hunk 1]
```diff
@@ -178,10 +173,8 @@ export default class CreationDateFacet extends React.PureComponent {
       </div>
     );
   }
-
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/apps/issues/sidebar/CreationDateFacet.js]: there are weird changes in this file (some lines are removed)

**Comment by gregaubert** [comment]
@stas-vilchik Okay so both bugs you showed me are fixed and I also fixed a problem with missing data that were not displayed on the preview graph of the project dashboard.

**Comment by stas-vilchik** [comment]
* there is an overflow problem on the project homepage
* there is a little problem when I move both handles to the very right

**Pull request merged**